src/test/java/org/jabref/logic/cleanup/EprintCleanupTest.java

```diff
@@ -27,4 +27,22 @@ void cleanupCompleteEntry() {
 
         assertEquals(expected, input);
     }
+
+    @Test
+    void cleanupEntryWithVersionAndInstitutionAndEid() {
+        BibEntry input = new BibEntry()
+                .withField(StandardField.NOTE, "arXiv: 1503.05173")
+                .withField(StandardField.VERSION, "1")
+                .withField(StandardField.INSTITUTION, "arXiv")
+                .withField(StandardField.EID, "arXiv:1503.05173");
+
+        BibEntry expected = new BibEntry()
+                .withField(StandardField.EPRINT, "1503.05173v1")
+                .withField(StandardField.EPRINTTYPE, "arxiv");
+
+        EprintCleanup cleanup = new EprintCleanup();
+        cleanup.cleanup(input);
+
+        assertEquals(expected, input);
+    }
```

Comment:
Please add another test with `INSTITUTION` `tbd`. That field value should be kept IMHO. --> JabRef should not destory data.

Reply:
I have made the suggested changes and pushed it to this PR , please review.

Reply:
I think, I was not clear with **another**

This means: A **second** test case. This means: Keep the original test as is.

---

I did not also not "unzip" my comment, sorry!

This will require changes in the code. The code needs to check if the institution is `arXiv`. Then, the field can be cleared. Otherwise, the field needs to be kept.